nixos/modules/module-list.nix

```diff
@@ -648,6 +648,7 @@
   ./services/hardware/buffyboard.nix
   ./services/hardware/ddccontrol.nix
   ./services/hardware/deepcool-digital-linux.nix
+  ./services/hardware/dell-bios-fan-control.nix
```

Comment:
Could you add a release note?

Reply:
sure I can do that. Forgive my ignorance, but which set of release notes does this belong in? Does this go straight into the newest (rl-2605.md)?